AXIOM_Remote_Firmware_Visualizer/VirtualUI.cpp

```diff
@@ -600,4 +609,4 @@ void VirtualUI::RenderUI(Button& button, int8_t& knobValue, bool& debugOverlayEn
     ImGui::EndFrame();
 
     ImGui::Render();
-}
+}
```

Comment:
Newline should be placed at the end of a file, some compilers will also warn about it or even fail, as it helps to find the end. Reference: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file